script/add-grammar

```diff
@@ -43,16 +43,24 @@ def log(msg)
   puts msg if $verbose
 end
 
-def command(*args)
+def command(*args, show_warnings: true)
   log "$ #{args.join(' ')}"
-  output, status = Open3.capture2e(*args)
-  if !status.success?
+  stdout, stderr, status = Open3.capture3(*args)
```

Comment:
UX nit: it's generally not a good idea to interleave STDERR and STDOUT unless the user is expected to do something with the latter (pipe to a pager, write the results to a file, copy it to the system's clipboard, etc).

Reply:
I'm not interleaving here, I'm explicitly splitting them. I merge them again later though 😁